.github/workflows/clang-format_check.yml

```diff
@@ -0,0 +1,10 @@
+name: clang-format check
+on: [push]
+jobs:
+  formatting-check:
+    name: Formatting Check
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Run clang-format style check for C programs.
+      uses: jidicula/clang-format-action@master
```

Comment:
Hi, I just want to point out that you're pointing to a nonexistent branch of my Action's repo, which is probably causing the workflow runs to fail. You can point instead to `main` or to a specific version tag (e.g. `@v3.2.0`)

Reply:
Thanks a lot for stopping by! We haven't ran the CI on this project in a while now, but it's definitely good to know. Thanks for your work on this

Reply:
Thanks a lot :heart: